lib/realms/zones/in_play.rb

```diff
@@ -1,26 +1,73 @@
 module Realms
   module Zones
+    class CardInPlay
+      attr_reader :card
+      attr_accessor :ally_activated, :base_activated
+
+      delegate_missing_to :card
```

Comment:
Interesting that this breaks `=== Realms::Cards::Card`, not that I should be doing that anyway...